docs/logging.md

```diff
@@ -22,3 +77,92 @@ code, time to process, (optional perf data details, e.g. database time/queries,
 memcached time/queries, Django process startup time, markdown processing time,
 etc.), URL, and "email via client" showing user account involved (if logged in)
 and the type of client they used ("web", "Android", etc.).
+
+The performance data details are particularly useful for investigating
+performance problems, since one can see at a glance whether a slow
+request was caused by delays in the database, in the markdown
+processor, in memcached, or in other Python code.
+
+One useful thing to note, however, is that the database time is only
+the time spent connecting to and receiving a response from the
+database.  Especially when response are large, there can often be a
+great deal of Python processing overhead to marshall the data from the
+database into Django objects that is not accounted for in these
+numbers.
```

Comment:
Hmm, in fact this graf gives me an idea to make the "format of this output" graf much better: it should be more tabular, perhaps bullets. Like

> The format of each line of this log is:
> * timestamp
> * log level
> [...]
> * optional perf data details, in "()": database time/queries, memcached time/queries, Django process startup time, markdown processing time, etc.
>   * These are particularly useful for investigating performance problems, since one can see at a glance whether a slow request was caused by delays in the database, in the markdown processor, in memcached, or in other Python code.
>   * One useful thing to note, however, is that the database time is only the time spent connecting to and receiving a response from the database. Especially when response are large, there can often be a great deal of Python processing overhead to marshall the data from the database into Django objects that is not accounted for in these numbers.
> * URL
> [...]

This way:
* it's easy to scan the list at a glance, rather than have to parse a lot of English to spot where one entry ends and the next begins;
* it's easy to add a paragraph or two of extra discussion on one item, and for it to be readily accessible right there when reading about that item.

Reply:
good idea, done.